setup.py

```diff
@@ -41,8 +41,10 @@ def read(*pathnames):
       install_requires=[
           'setuptools',
           'zc.queue >= 1.3',
-          'requests < 1.0.0',
+          'requests',
           'plone.z3cform',
+          'plone.api',
```

Comment:
What's the value added of depending on `plone.api`. Yes, some community members seem to favor yet another indirection, but Plone actually does have an API and `getToolByName` is part of it.

Or via acquisition, simply `self.context.<some_tool>`.